ix-dev/test/glances/templates/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set glances_container = tpl.add_container(values.consts.container_name, "image") %}
+
+{% do glances_container.add_caps(["FOWNER", "DAC_OVERRIDE", "SETGID", "SETUID", "SYS_PTRACE"]) %}
+{% do glances_container.remove_security_opt("no-new-privileges") %}
+{% do glances_container.add_security_opt("apparmor=unconfined") %}
+
+{% do glances_container.healthcheck.disable() %}
+
+{% if values.glances.server_enable %}
+  {% do glances_container.environment.add_env("GLANCES_OPT", "-w") %}
+  {% do glances_container.ports.add_port(61208, 61208) %}
+  {% do tpl.portals.add_portal({"name": "System Overview", "port": 61208, "path": "/"},) %}
+{% endif %}
+
+{% if values.glances.docker_enable %}
+  {% do glances_container.add_docker_socket(read_only=True) %}
+{% endif %}
+
+{% do glances_container.add_storage("/host/etc/os-release", {"type": "host_path", "read_only": true, "host_path_config": {"path": "/etc/os-release"}}) %}
+{% do glances_container.add_storage("/host/sys", {"type": "host_path", "read_only": true, "host_path_config": {"path": "/sys"}}) %}
+{% do glances_container.add_storage("/host/proc", {"type": "host_path", "read_only": true, "host_path_config": {"path": "/proc"}}) %}
+{% do glances_container.add_storage("/host/etc/passwd", {"type": "host_path", "read_only": true, "host_path_config": {"path": "/etc/passwd"}}) %}
+{% do glances_container.add_storage("/host/etc/group", {"type": "host_path", "read_only": true, "host_path_config": {"path": "/etc/group"}}) %}
+
+{% do glances_container.set_tty(true) %}
+{% do glances_container.set_stdin(true) %}
+
+{% do glances_container.environment.add_user_envs(values.glances.additional_envs) %}
```

Comment:
move those under the remove/add sec opts, so its consistent with other apps ordering